### PR TITLE
Hit api PR

### PR DESCRIPTION
**Pull request by adamteddychang**
Features Implemented:
- created a new game to generate a unique id
- used async and await JavaScript features to consume API
- implemented refresh button to receive data from the API
- implemented submit button to send data from form to API

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      scores.innerHTML = '';
+      if (scoresArr.length > 0) {
+        scoresArr.forEach((e) => {
+          const scoresTemp = `<li class="score" >${e.user}: ${e.score}</li>`;
+          scores.innerHTML += scoresTemp;
+        });
+      }
+    }
+  });
+};
```
Comment (henatan99) [inline src/api.js]: - [x] Kindly use async/await for asynchronous operations as it is the project's requirement. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @adamteddychang , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/api.js (hunk 1 above)
Comment (adamteddychang) [reply src/api.js]: I have added async/await to that function, thank you.

Comment (henatan99) [reply src/api.js]: Well done! This issue is resolved. :clap: :clap: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @adamteddychang , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/api.js (hunk 1 above)
Comment (adamteddychang) [reply src/api.js]: Thank you!

**Pull request merged**